pkg/controller/deployment/deployment_controller.go

```diff
@@ -63,9 +63,9 @@ const (
 // controllerKind contains the schema.GroupVersionKind for this controller type.
 var controllerKind = apps.SchemeGroupVersion.WithKind("Deployment")
 
-// DeploymentController is responsible for synchronizing Deployment objects stored
+// Controller is a controller for Deployments, is responsible for synchronizing Deployment objects stored
 // in the system with actual running replica sets and pods.
-type DeploymentController struct {
+type Controller struct {
```

Comment:
I'm not sure if it would be appropriate to rename `DeploymentController` to `Controller` ，
I noticed how other controllers named in their package, only `cronjob` package named `Controller`.  Others such as `job`,  `daemon`,  `replicaset`...are all named `${packageName}Controller`.

Reply:
I'm very glad to modify other packages if this pr works.

Reply:
The filename `deployment_controller.go` feels like a strong hint too.

Reply:
> The filename `deployment_controller.go` feels like a strong hint too.

What do you think of if filename rename to `controller.go`?

Reply:
> Others such as job, daemon, replicaset...are all named ${packageName}Controller.

I'd leave this one as `${packageName}Controller` too.

Reply:
ok...think this golint error not easy to fix,  close pr?